data/ep-blueprint-data-bellevie_8.0.0/liquibase-changelog.xml

```diff
@@ -3,9 +3,9 @@
   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd"
   logicalFilePath="ep-blueprint-data/liquibase-changelog.xml">
 
-	<changeSet id="vestri-data-import" author="elasticpath" context="ep-blueprint">
+	<changeSet id="bellevie-data-import" author="elasticpath" context="ep-blueprint">
```

Comment:
Let's not delete/overwrite the existing vestri data. We should keep the expanded vestri data under `ep-blueprint-data_8.0.0` and add the new BelleVie data in a new directory `ep-blueprint-data-bellevie_8.0.0` so the two data sets can co-exist in our repo